src/mongocxx/options/find.hpp

```diff
@@ -128,6 +129,28 @@ class MONGOCXX_API find {
     const stdx::optional<std::int32_t>& limit() const;
 
     ///
+    /// The maximum amount of time for the server to wait on new documents to satisfy a tailable
+    /// cursor query. This only applies to a TAILABLE_AWAIT cursor. When the cursor is not a
+    /// TAILABLE_AWAIT cursor, this option is ignored. The default is no value.
```

Comment:
A bit confusing, reword to something like "The default is to wait until the operation completes"